stable/jenkins/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 name: jenkins
 home: https://jenkins.io/
-version: 0.16.24
+version: 0.16.25
```

Comment:
new feature - so I'd say 0.17.0

Reply:
@lsowen Ping?

Reply:
@davidkarlsen sorry, holiday and all that.  I'll update it now.